main.tf

```diff
@@ -42,6 +42,11 @@ resource "aws_redshift_cluster" "this" {
   preferred_maintenance_window        = var.preferred_maintenance_window
   allow_version_upgrade               = var.allow_version_upgrade
 
+  snapshot_copy {
```

Comment:
AFAIK `snapshot_copy` block should be optional also if the user does not want to copy snapshots.

Please upgrade to use `dynamic` block for this.

Reply:
I think if the `destination_region` is not specified (default empty string), snapshots will not be copied. But, the dynamic block should be a way to go probably.

Reply:
Done.